Close #335 by fixing typos and removing instances of passive voice

hamirmahal opened the pull request:
This pull request seeks to close #335 by changing "hello work" to "hello world" in [the "Examples" section of `README.md`](https://github.com/amilajack/alfred/blob/master/README.md#examples) and [`examples/README.md`](https://github.com/amilajack/alfred/blob/master/examples/README.md), and changing a few typos and a couple of instances of the passive voice in [`website/docs/configuration.md`](https://github.com/amilajack/alfred/blob/master/website/docs/configuration.md), as mentioned in [the "Guidelines for Writing Docs" section in `CONTRIBUTING.md`](https://github.com/amilajack/alfred/blob/master/CONTRIBUTING.md#guidelines-for-writing-docs).

hamirmahal committed:
close #335 by fixing typos to improve readability

This commit also changes a couple instances of the passive voice to "Never use the passive where you can use the active", as stated in this repository's CONTRIBUTING.md file.
https://github.com/amilajack/alfred/blob/master/CONTRIBUTING.md#guidelines-for-writing-docs

hamirmahal committed:
change "hello work" to "hello world" in 2 READMEs

This commit changes "hello work" to "hello world" in examples/README.md and README.md.

amilajack commented on README.md:
```diff
@@ -44,7 +44,7 @@ See our [examples directory](https://github.com/amilajack/alfred/tree/master/exa
 |     | Example                                 | Descrption                              |
 |-----|-----------------------------------------|-----------------------------------------|
-| 1.  | [hello world node][hello-world-example] | A simple hello work app in node         |
+| 1.  | [hello world node][hello-world-example] | A simple hello world app in node         |
```
LOOL "work" 😆

hamirmahal replied:
Ha! I thought it was a typo.

amilajack commented on website/docs/configuration.md:
````diff
@@ -68,7 +68,7 @@ Suppose your Alfred project has the following config:
 }
 ```
 
-This config can be extracted to an Alfred config by creating a sharable config. In this case, it makes sense to call the config `alfred-config-web-app`.
````
Oh wow this was definitely super confusing haha. Good catch!

hamirmahal replied:
Thank you!

amilajack commented on website/docs/configuration.md:
```diff
@@ -39,19 +39,19 @@ The skills that the Alfred project is using
 Default: `.`
 
-The directory, relative to the project directory, which config files should be written to
+The directory, relative to the project directory, to write config files
```
What do you think about this:

```suggestion
The path (relative to project directory) that config files are written to
```

hamirmahal replied:
I think that looks good! [I committed your suggestion](https://github.com/amilajack/alfred/pull/336/commits/8e8aa9c0292c94df6f4599e0b6122ec7b8f41834).

hamirmahal committed:
Improve readability of website/docs/configuration

This commit changes "The directory, relative to the project directory, to write config files" to "The path (relative to project directory) that config files are written to" to improve readability.

Co-authored-by: Amila Welihinda <amilajack@gmail.com>

amilajack commented on website/docs/configuration.md:
````diff
@@ -68,7 +68,7 @@ Suppose your Alfred project has the following config:
 ```
 
-This config can be extracted to an Alfred config by creating a sharable config. In this case, it makes sense to call the config `alfred-config-web-app`.
+Creating a shareable config will extract this config to an Alfred config. In this case, it makes sense to call the config `alfred-config-web-app`.
````
how about
```suggestion
Creating a shareable config will extract this config into an Alfred config. In this case, it makes sense to call the config `alfred-config-web-app`.
```
?

hamirmahal replied:
a08c97c makes this change.

amilajack reviewed (commented):
Some minor suggestions

hamirmahal committed:
Improve readability for website/docs/configuration

This commit changes "Creating a shareable config will extract this config to an Alfred config. In this case, it makes sense to call the config `alfred-config-web-app`" to "Creating a shareable config will extract this config into an Alfred config. In this case, it makes sense to call the config `alfred-config-web-app`".

Co-authored-by: Amila Welihinda <amilajack@gmail.com>

amilajack reviewed (approved):
Thanks for these fixes Hamir!

hamirmahal commented:
You're welcome, Amila!

I'm happy to help.
